public/index.html

```diff
@@ -145,15 +145,19 @@ <h3 class="panel-title">Proposals</h3>
                  <td>Hash Race Exchange Listing #1</td>
                  <td>5 BTC Fund Raiser</td>
                  <td>In progress <span class="label label-danger">New</span></td>
-                 <td><a href="https://hodler95.github.io/exchangesrace/" target="_blank">Race Tracker / </a><a href="https://docs.google.com/document/d/1R3E_XQdR4EaeSdoqD5w-o7ieL9y-_kgqBQz0j8gKOlk/edit?usp=sharing" target="_blank">Race Details </a></td>
+                 <td>
+                     <a href="https://doodle.com/poll/t2bpenn288s6ndna" target="_blank">Vote now! / </a>
+                     <a href="https://hodler95.github.io/exchangesrace/" target="_blank">Race Tracker / </a>
+                     <a href="https://docs.google.com/document/d/1R3E_XQdR4EaeSdoqD5w-o7ieL9y-_kgqBQz0j8gKOlk/edit?usp=sharing" target="_blank">Race Details </a>
+                 </td>
                 </tr>
-               </tbody>                  
+               </tbody>
```

Comment:
FYI -- all these changes below were done by my vim configs automatically (it removes trailing whitespace, etc). I was too lazy to undo.